js/vimlparser.js

```diff
@@ -1864,7 +1864,7 @@ VimLParser.prototype.parse_cmd_catch = function() {
     if (!this.ends_excmds(this.reader.peek())) {
         var __tmp = this.parse_pattern(this.reader.get());
         node.pattern = __tmp[0];
-        var endc = __tmp[1];
+        var _ = __tmp[1];
```

Comment:
Sorry, i don't understand what you said. Significance?

Did you mean we should remove this assignment by JavaScript compiler?